src/QuantumSdk/DefaultItems/DefaultItems.targets

```diff
@@ -44,17 +44,18 @@
       These architectures are currently under active development and not available for public use yet. 
       Targeting is disabled since the code will be targeted by the QIR compiler, and Q# diagnostics are not yet enabled for these architectures. 
     -->
-    <ResolvedProcessorArchitecture Condition="('$(ResolvedQSharpOutputType)' == 'QSharpExe' Or '$(NoEntryPoint)' == 'true') And $(ExecutionTarget.ToLowerInvariant().StartsWith('honeywell.qir'))">HoneywellQirProcessor</ResolvedProcessorArchitecture>
-    <ResolvedProcessorArchitecture Condition="('$(ResolvedQSharpOutputType)' == 'QSharpExe' Or '$(NoEntryPoint)' == 'true') And $(ExecutionTarget.ToLowerInvariant().StartsWith('qci.qir'))">QCIQirProcessor</ResolvedProcessorArchitecture>
+    <!-- TODO: This might be temporary while it is decided how QIR targets are going to be handled. -->
+    <ResolvedProcessorArchitecture Condition="('$(ResolvedQSharpOutputType)' == 'QSharpExe' Or '$(NoEntryPoint)' == 'true') And $(ExecutionTarget.ToLowerInvariant().StartsWith('quantinuum'))">QuantinuumQirProcessor</ResolvedProcessorArchitecture>
+    <ResolvedProcessorArchitecture Condition="('$(ResolvedQSharpOutputType)' == 'QSharpExe' Or '$(NoEntryPoint)' == 'true') And $(ExecutionTarget.ToLowerInvariant().StartsWith('qci'))">QCIQirProcessor</ResolvedProcessorArchitecture>
```

Comment:
Actually, before merging, we need to make sure the mechanism for opting into QIR is well understood. As written, these lines will override for any Quantinuum or QCI targets and force usage of the target packages below, which may not work with the existing service translation logic.

Reply:
Yes, we do need to do that before merging into `main` but this is currently targeting a feature branch.
@bettinaheim, you mentioned that you were working on something that also needed SDK changes that was going to be merged into a feature branch. Please let me know if you want this PR to target a different branch.

Reply:
Let's merge this into the feature branch, and I'll update the Sdk handling to have both coexist peacefully as part of also propagating the necessary information into the compiler.

Reply:
ah, good point, I didn't notice it was to a feature branch!